Content.Server/VendingMachineRestockPackage/VendingMachineRestockPackageComponent.cs

```diff
@@ -0,0 +1,38 @@
+using System.Threading;
+using Robust.Shared.Audio;
+
+namespace Content.Server.VendingMachineRestockPackage
+{
+    [RegisterComponent]
+    public sealed class VendingMachineRestockPackageComponent : Component
+    {
+        public CancellationTokenSource? CancelToken;
+
+        /// <summary>
+        /// The time (in seconds) that it takes to restock a machine.
+        /// </summary>
+        [DataField("restockDelay")]
+        public float RestockDelay = 8.0f;
+
+        /// <summary>
+        /// What sort of machine inventory does this restock?
+        /// This is checked against the VendingMachineComponent's pack value.
+        /// </summary>
+        [DataField("canRestock")]
+        public HashSet<string> CanRestock = new();
+
+        /// <summary>
+        ///     Sound that plays when starting to restock a machine.
+        /// </summary>
+        [DataField("soundRestockStart")]
+        // Credit to: https://freesound.org/people/Defaultv/sounds/534362/
```

Comment:
these shoudl be attributed in the folder with the audio files themselves, under an `attributions.yml` RGA format; there are a couple examples of this you can copy